docs/user/how-to/run-cli.rst

```diff
@@ -0,0 +1,80 @@
+Control the Worker via the CLI
+==============================
+
+The worker comes with a minimal CLI client for basic control. It should be noted that this is 
+a test/development/debugging tool and not meant for production!
+
+.. seealso:: 
+    You must have access to the ``blueapi`` command either via the container or pip installing.
+    `./run-container` and `../tutorials/installation`
+
+
+.. seealso:: 
+    In a developer environment, the worker can also be run from vscode: `../../developer/tutorials/dev-run`.
+
+
+Start ActiveMQ
+--------------
+
+The worker requires a running instance of ActiveMQ, the simplest
+way to start it is to run it via a container:
+
+.. tab-set::
+
+    .. tab-item:: Docker
+
+        .. code:: shell
+
+            docker run -it --rm --net host rmohr/activemq:5.15.9-alpine
+
+    .. tab-item:: Podman
+
+        .. code:: shell
+
+            podman run -it --rm --net host rmohr/activemq:5.15.9-alpine
+
+
+Start the Worker
+----------------
+
+To start the worker
+
+.. code:: shell
+
+    blueapi serve
+
+
+Basic Introspection
+-------------------
+
+The worker can tell you which plans and devices are available via:
+
+.. code:: shell
+
+    blueapi controller plans
+    blueapi controller devices
+
+By default, the CLI will talk to the worker via a message broker on ``tcp://localhost:61613``,
+but you can customize this.
+
+.. code:: shell
+
+    blueapi controller -h my.host -p 61614 plans
+
+Running Plans
+-------------
+
+You can run a plan and pass arbitrary JSON parameters.
+
+.. code:: shell
+
+    # Run the sleep plan
+    blueapi controller run sleep -p '{"time": 5.0}'
+
+    # Run the count plan
+    blueapi controller run count -p '{"detectors": ["current_det", "image_det"]}'
+
+The command will block until the plan is finished and will forward error/status messages 
```

Comment:
The run command no longer blocks or forwards status messages

Reply:
Hmm, it should, will restore that functionality...

Reply:
In future the CLI run command will be blocking.

Reply:
Now done